kotest-framework/kotest-framework-engine/src/jvmMain/kotlin/io/kotest/engine/KotestEngine.kt

```diff
@@ -75,12 +77,14 @@ class KotestEngine(private val config: KotestEngineConfig) {
                   )
                   return
                }
-
-
             }
          )
 
-      Try { submitAll(plan) }
+      Try {
+         runProjectExtensions(projectExtensions) {
```

Comment:
After thinking about this more I'm going to move the try inside so the tests that explode still allow the end of the interceptor to complete

Reply:
Or do we put the responsibility to catch and re-throw on the extension implementation

Reply:
I would gather the errors and then fail the test run at the end.